internal/server/v1/page/golden/place_page/asm.Energy.json

```diff
@@ -819,11 +819,12 @@
       }
     },
     "country/NZL": {
-      "date": "2022",
-      "value": 5151600,
+      "date": "2021",
+      "value": 5111400,
       "metadata": {
-        "importName": "New_Zealand_Census",
-        "provenanceUrl": "https://www.stats.govt.nz/"
```

Comment:
We might perhaps add the per-country censuses ahead of WDI and Eurostat too

Reply:
That would be good. @ajaits can suggest some imports?